cmd/utils.go

```diff
@@ -626,6 +626,11 @@ func getGitLabels(config *RootCommandConfig) (map[string]string, error) {
 		labels[ociKeyPrefix+"url"] = gitInfo.RemoteURL
 		labels[ociKeyPrefix+"documentation"] = gitInfo.RemoteURL
 		labels[ociKeyPrefix+"source"] = gitInfo.RemoteURL + "/tree/" + gitInfo.Branch
+		upstreamSplit := strings.Split(strings.Split(gitInfo.Upstream, " ")[0], "/")
```

Comment:
Just wondering why are we spliting the upstream by whitespaces? From https://github.com/appsody/appsody/blob/master/cmd/git_utils.go#L165, we just split on `/` since the format should be `upstream/branch`.

Reply:
Because the gitInfo.Upstream may contain a space and then more information. For example:
```
$ git status -sb
## useUpstreamBranchInGitUrls...kylegc/useUpstreamBranchInGitUrls [ahead 1]
```
I need to first strip off the `" [ahead 1]"` Then split on the `/`.

Perhaps this should have been done before gitInfo.Upstream was set?

Reply:
issue created https://github.com/appsody/appsody/issues/554